my-site.rb

```diff
@@ -5,6 +5,10 @@ class MySite < Sinatra::Base
   register Sinatra::Reloader
 
   get "/" do
+    send_file 'public/index.html'
+  end
 
+  get "/" do
```

Comment:
A single concern here--how is this route different from the route on line 7? What's the instruction you'd need to give Sinatra to have it respond with the projects.html?

To put it another way--what would happen if you moved your html files from `/public` to `/static`?